index.html

```diff
@@ -10,10 +10,59 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <img src="img/karma-logo.svg" alt="Logo">
+        <nav>
+            <ul>
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+ 
+    <main>
+        <div class="hero">
+            <img src="img/first-background.jpg">
+            <div class="hero-text">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.<h2>
```

Comment:
```suggestion
            <p>Bring WiFi with you, everywhere you go.<p>
```
This doesn't seem like a heading to me — there is already a heading above it — so we could change this to a `p`